src/modules/tools/endstops/Endstops.cpp

```diff
@@ -367,8 +368,8 @@ void Endstops::move_to_origin(char axes_to_move)
     // Move to center using a regular move, use slower of X and Y fast rate
     float rate = std::min(this->fast_rates[0], this->fast_rates[1]) * 60.0F;
     char buf[32];
-    snprintf(buf, sizeof(buf), "G53 G0 X0 Y0 F%1.4f", rate); // must use machine coordinates in case G92 or WCS is in effect
     THEKERNEL->robot->push_state();
+	snprintf(buf, sizeof(buf), "G21 G53 G0 X0 Y0 F%1.4f", rate); // must use mm units & machine coordinates in case G92 or WCS is in effect
```

Comment:
Does this really need to be in inch mode? isn't X0Y0 the same regardless of mm or inch? 0 is zero right?